test/e2e/workflow/deployment_test.go

```diff
@@ -165,6 +165,9 @@ var _ = Describe("NetworkAddonsConfig", func() {
 		//2300
 		It("should be able to remove the config and create it again", func() {
 			DeleteConfig()
+			//TODO: remove this checking after this [1] issue is resolved
```

Comment:
not sure why this commit is in this PR. can you explain why not in separate?

Reply:
Without this check the e2e-workflow lane fails